include/runtime/header.h

```diff
@@ -130,6 +132,8 @@ void hash_exit(void);
 
 class KElem {
 public:
+  KElem() { this->elem = NULL; }
```

Comment:
what is this constructor used for?

Reply:
This constructor was added because I included the constructor `Range()` ([here](https://github.com/runtimeverification/llvm-backend/blob/5330cee8a1833256aa3e64d9b26a7a5900159194/include/runtime/collections/rangemap.h#L32)), that creates an empty `Range`. That was needed in the implementation of a function that needs to construct `Range`s from `KElem`s, [here](https://github.com/runtimeverification/llvm-backend/blob/5330cee8a1833256aa3e64d9b26a7a5900159194/include/runtime/collections/RBTree.h#L385).